### PR TITLE
Add default values of backend args to info endpoint

### PR DESCRIPTION
**Pull request by LefterisJP**

### CODE REVIEW

File: docs/api.rst [hunk 1]
```diff
@@ -10465,7 +10465,12 @@ Querying general information
...
               "data_directory": "/home/username/.local/share/rotki/data"
               "log_level": "DEBUG",
-	      "accept_docker_risk: false
+	      "accept_docker_risk: false,
+	      "backend_default_arguments": {
+	          "max-logfiles-num": 3,
+	          "max-size-in-mb-all-logs": 300,
+	          "sqlite-instructions": 5000,
+	      },
           },
```
Comment (yabirgb) [inline docs/api.rst:10474]: maybe we can change this in other PR since is green but the trailing `,` is something kelsos complains about

**Review by yabirgb — APPROVED** [review]
just a small comment that can be changed in other PR

File: docs/api.rst (hunk 1 above)
Comment (LefterisJP) [reply docs/api.rst:10474]: It's not just kelsos. It's the docs failing on invalid json. Weird it did not fail.

Comment (LefterisJP) [reply docs/api.rst:10474]: I will change in another docs only PR though

**Pull request merged**